infra/src/main/java/com/redhat/service/bridge/infra/k8s/ResourceEvent.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.service.bridge.infra.k8s;
+
+public class ResourceEvent {
+
+    private String subject;
+
+    private String resourceId;
+
+    private Action action;
+
+    public ResourceEvent(String subject, String resourceId, Action action) {
```

Comment:
Just to make sure I understand, `subject` here is the `type` e.g `processor` or `bridge`? If yes, can we rename to `type` for consistency.